### DISCUSSION TITLE
Delay Option

### DISCUSSION THREAD

**Discussion by TheCodeGeek2** [porrey/Virtual-ZPL-Printer#17] [General]
Hello Daniel,

I was having a problem with the application not getting all of the labels due to the restrictions on Labelary's API. Specifically the 5 labels per second limit. I looked into the code after forking it and found that you had already started working on this, or so it would appear. I'm talking about the LingerOption in Modified LingerOption in TcpListenerClientHandler.cs. I tried setting this to "true, 1" and it was able to get all of my labels. Are you planning on adding this to an options menu? It would be of great help. Alternatively, I can't figure out how to build the installer, I keep getting an error and it fails with:
ERROR: General failure building project output groups
ERROR: Unrecoverable build error - 0x80004005
When I run VS as Admin, It tries a little harder but then still gets errors and refuses to build. I've been able to find a workaround by just copying the files to the app directory in C: and it will run fine, but I would like to be able to roll up the installer.

**Comment by porrey** [comment]
I will consider this in the next release.

**Comment by porrey** [comment]
See release 2.3.0
